apps/nextjs-website/react-components/types/NotFoundPage/NotFoundPage.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactElement } from 'react';
+import { SectionProps } from '@react-components/types/common/Common.types';
+
+export interface NotFoundPageProps extends SectionProps {
+  image: ReactElement;
+  mobileImage: ReactElement;
+  redirectUrl?: string;
+  disableRedirect?: boolean;
+  locale?: 'it' | 'en' | 'fr' | 'de' | 'sl';
```

Comment:
```suggestion
  locale: 'it' | 'en' | 'fr' | 'de' | 'sl';
```
Let's make this required to make it harder to forget to set the field.